predict_pv_yield/models/base_model.py

```diff
@@ -1,8 +1,11 @@
 import pytorch_lightning as pl
 import torch
 import torch.nn.functional as F
-from predict_pv_yield.visualisation import plot_example
+from predict_pv_yield.visualisation.visualisation import plot_example
+from predict_pv_yield.visualisation.line import plot_batch_results
+from nowcasting_dataset.data_sources.nwp_data_source import NWP_VARIABLE_NAMES
```

Comment:
I can image this changing, and then we might have to load form data config file - but i thought for the moment this is ok

Reply:
Sure, I agree, best to keep it simple for now!